### PR TITLE
fix creation containers on btrfs subvolumes

### PR DESCRIPTION
**Pull request by sapun**

### CODE REVIEW

**Comment by gregdek** [comment]
@cloudnull Please review this PR to make sure it adheres to the following guidelines:

http://docs.ansible.com/developing_modules.html#module-checklist

If it passes these guidelines, and if you believe it’s a good PR otherwise, please add a comment with "shipit" in the text, and we will flag it for inclusion.

If it fails these guidelines, please add a comment with "needs_revision" in the text, along with the needed changes, and we will flag it for revision.

File: cloud/lxc/lxc_container.py [hunk 1]
```diff
@@ -448,7 +448,7 @@
         'zfs_root'
     ],
     'btrfs': [
-        'lv_name', 'vg_name', 'thinpool', 'zfs_root'
+        'lv_name', 'vg_name', 'thinpool', 'zfs_root', 'fs_type', 'fs_size'
```
Comment (cloudnull) [inline cloud/lxc/lxc_container.py:451]: Quick question here but does it really need to NOT have storage type and size when using a btrfs container?  I've not seen that though admittedly I've not tested btrfs in some time. If so, the change lgtm but I'd like to know why this change is required first.

Comment (sapun) [inline cloud/lxc/lxc_container.py:451]: Because without this fix i catch error like this
"err": "filesystem type and size are only valid with block devices\n"

Comment (cloudnull) [inline cloud/lxc/lxc_container.py:451]: fair enough.

**Comment by cloudnull** [comment]
@gregdek  shipit - LGTM!

**Comment by gregdek** [comment]
Thanks @sapun for the patch, and @cloudnull for the review. Marked for inclusion.
